pkgs/by-name/ro/rollo-x1038/package.nix

```
@@ -0,0 +1,34 @@
+{
+  stdenv,
+  cups,
+  lib,
+  fetchzip,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "rollo-x1038";
+  version = "1.8.4";
+  src = fetchzip {
+    url = "https://rollo-main.b-cdn.net/driver-dl/linux/rollo-cups-driver-${finalAttrs.version}.tar.gz";
+    hash = "sha256-BJ6yj59MiLMfbEkf+Ud4ypDWgskxjrXHJCfZIrlpDXU=";
+  };
+  nativeBuildInputs = [
+    cups
+  ];
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm755 rastertorollo -t $out/lib/cups/filter
+    install -Dm644 rollo-x1038.ppd -t $out/share/cups/model
+
+    runHook postInstall
+  '';
```

Ight, I found a solution in [another similar driver package](https://github.com/NixOS/nixpkgs/blob/eb35e32444d75a72c989be2324a4f4f706921515/pkgs/by-name/li/libppd/package.nix#L44-L48):
```suggestion
  nativeBuildInputs = [
    pkg-config
    cups
  ];
  buildInputs = [
    cups
  ];
  makeFlags = [
    "CUPS_SERVERBIN=$(out)/lib/cups"
    "CUPS_DATADIR=$(out)/share/cups"
    "CUPS_SERVERROOT=$(out)/etc/cups"
  ];

```

---

Now the original `make install` target works properly (initially it wanted to create new dirs inside `cups`, and `./configure` doesn't have an option for that)